adjust_binning.py

```diff
@@ -380,19 +380,19 @@ def get_merged_cuts(
     for process in args.processes:
         console.print(f"\n[bold green]Processing: {process}[/bold green]")
 
-        if process == "process_fractions":
-            if "process_fractions" not in config:
-                console.print("[red]Block 'process_fractions' not found in config. Skipping.[/red]")
+        if process == "process_fractions" or process == "process_fractions_subleading":
```

Comment:
`if process in ["process_fractions", "process_fractions_subleading"]:`
or `if process.find("process_fractions") >=0:` to make it shorter.